libraries/AP_Soaring/SoaringController.cpp

```diff
@@ -0,0 +1,407 @@
+#include "SoaringController.h"
+
+extern const AP_HAL::HAL& hal;
+
+
+// ArduSoar parameters
+const AP_Param::GroupInfo SoaringController::var_info[] = {
+    // @Param: ACTIVE
+    // @DisplayName: Is the soaring mode active or not
+    // @Description: Toggles the soaring mode on and off
+    // @Units: boolean
+    // @Range: 0 1
+    // @User: Advanced
+    AP_GROUPINFO("ACTIVE", 1, SoaringController, soar_active, 0),
```

Comment:
What if we made ACTIVE an ENABLE parameter? Or is this considered the way a user toggles soaring on/off while in flight and hiding the rest of the parameters would be a bad idea.